versions/draft-3/SPEC.md

```diff
@@ -636,6 +637,16 @@ $document = ($import | $task | $workflow)+
 
 `$document` is the root of the parse tree and it consists of one or more import statement, task, or workflow definition
 
+## Versioning
+
+For portability purposes it is critical that WDL documents be versioned so an engine knows how to process it. A WDL document is versioned with a `version` statement, for eample
+
+```wdl
+version draft-3
+```
+
+All WDL documents must have exactly one `version` statement. Any WDL document which does not have a `version` field must be treated as `draft-2`.  
```

Comment:
You know I hate being annoyingly pedantically, but... 

The second sentence here now falsifies the first. It's probably clearer to say `"From draft-3 onwards, all WDL documents must ..."`